main.py

```diff
@@ -7,20 +8,42 @@
 
 from blogReader import APIChecker, msg_new_article, build_embed
 
+# Set up global variables
 BOT_TOKEN = getenv('BOT_TOKEN')
 CHANNEL_ID = getenv('CHANNEL_ID')
+REFRESH_DELAY = int(getenv('REFRESH_DELAY')) #in minutes
 
 # LOG_LEVEL can either be an int or a string
 LOG_LEVEL = getenv('LOG_LEVEL')
 if LOG_LEVEL.isnumeric():
     LOG_LEVEL = int(LOG_LEVEL)
 
+# Set up logging
 discord.utils.setup_logging(root=True, level=LOG_LEVEL)
 
+def log2file_handler():
+    log_file = getenv('LOG_FILE')
+    if not log_file:
+        return None
+
+    logs_dir = path.dirname(log_file)
```

Comment:
we don't want to perform checks on the parent dir of the log file but on the log file itself, it must have been created by run.sh and have the right permisssions so it can't be created on the fly by FileHandler!

Reply:
I was wrong, because the FileHandler opens the file in `a` mode it creates it if it doesn't already exists, we must only check the parent dir.

Ofc, it is, in reality, ensured by docker mounting the volume but we must take care of users deployign without it